src/files.h

```diff
@@ -140,7 +140,8 @@ R"RAW(
 #endif
 R"RAW(
         "close_tab": "<primary>w",
-        "window_toggle_split": ""
+        "window_toggle_split": "",
+        "clear_console": "<primary><shift>c"
```

Comment:
Maybe window_clear_terminal? Since we use terminal in the source code, and elsewhere.